src/guides/v2.4/graphql/mutations/create-customer-v2.md

```diff
@@ -0,0 +1,101 @@
+---
+group: graphql
+title: createCustomerV2 mutation
+---
+
+The `createCustomerV2` mutation creates a customer account. Use the [`createCustomerAddress` mutation]({{page.baseurl}}/graphql/mutations/create-customer-address.html) to complete the customer profile and define billing and shipping addresses.
+
+The `createCustomerV2` mutation replaces the `createCustomer` mutation as the means to create a customer account. The input objects differ between these two mutations. The `createCustomer` mutation required the `CustomerInput` object, as did the `updateCustomer` mutation. The attributes required for creating a customer are different than those for updating a customer, but you could not determine this by looking at the schema. The `createCustomerV2` mutation requires the `CustomerCreateInput` object, which it does not share with the [`updateCustomerV2` mutation]({{page.baseurl}}/graphql/mutations/update-customer-v2.html).
+
+## Syntax
+
+`mutation: {createCustomerV2(input: CustomerCreateInput!) {CustomerOutput}}`
+
+## Example usage
+
+The following call creates a new customer.
+
+**Request:**
+
+```graphql
+mutation {
+  createCustomerV2(
+    input: {
+      firstname: "Bob"
+      lastname: "Loblaw"
+      email: "bobloblaw@example.com"
+      password: "b0bl0bl@w"
+      is_subscribed: true
+    }
+  ) {
+    customer {
+      firstname
+      lastname
+      email
+      is_subscribed
+    }
+  }
+}
+```
+
+**Response:**
+
+```json
+{
+  "data": {
+    "createCustomer": {
+      "customer": {
+        "firstname": "Bob",
+        "lastname": "Loblaw",
+        "email": "bobloblaw@example.com",
+        "is_subscribed": true
+      }
+    }
+  }
+}
+```
+
+## Input attributes
+
+The following table lists the attributes you can use as input for the `createCustomerV2` mutation.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`date_of_birth` | String | The customer’s date of birth
+`dob` | String | Deprecated. Use `date_of_birth` instead. The customer’s date of birth
```

Comment:
should we reference this "dob" here? maybe just add a note that dob was deprecated from CustomerCreateInput and CustomerUpdateInput

Reply:
Right now, all attributes are listed alphabetically. I will consider making a sweep through all the GraphQL docs and de-emphasize deprecated attributes, but that has to be done globally. Otherwise, we'll get a lot of PRs with people re-alphabetizing or re-ordering them.